### DISCUSSION TITLE
Add managed package lists

### DISCUSSION THREAD

**Discussion by tbcahill** [Devolutions/UniGetUI#1676] [General]
Add the option to point WingetUI to a custom package list URL.
This would allow an admin to curate/export a list of apps to install & then host that list - agents that are pointed to that list will automatically install the packages that are listed. 

It would be great to see this tool mirror some of Munki's management features ...

**Comment by marticliment** [comment]
You can just create custom scoop buckets and then add those buckets to WingetUI, which also allows updates from your custom endpoint.

Custom chocolatey and winget sources are also available, and therefore they can also be used as an approach to have custom package lists installed and updated by WingetUI

**Reply by tbcahill** [reply]
Scoop buckets might work for force installing apps. Thanks
I was thinking more allong the lines of how Munki's repo works where you would use scoop like manifests to tell WingetUI what  applications (winget/chocolatey/other sources) are mandatory & what ones are optional. That way WingetUI acts as a internal app store containing only approved apps.
Munki does this really well for Mac but I've been struggeting to find an equivelent for Windows

**Reply by marticliment** [reply]
What you could do with Scoop to force install apps is creating an _empty_ scoop package, and then listing all the packages you want to install as dependencies for this dummy package. Then, you will just need to install the mega-package (conveniently named "Package Pack") and scoop will install the dependencies (aka the actual programs you want to install). Then, to update the list of installed packages for your users you just need to update the mega-package with new dependencies or removing unneeded ones.

You can find more info on how to use dependencies here: https://github.com/ScoopInstaller/Scoop/wiki/Dependencies
